low-level/files/opt/seagate/sspl/setup/sspl_reset.py

```diff
@@ -2,89 +2,56 @@
 import shutil
 
 from framework.base.sspl_constants import (PRODUCT_FAMILY,
-                                           file_store_config_path,
-                                           SSPL_CONFIGURED,
                                            DATA_PATH,
                                            SSPL_CONFIG_INDEX,
                                            sspl_config_path)
-from cortx.utils.process import SimpleProcess
 from cortx.utils.service import DbusServiceHandler
 from cortx.utils.conf_store import Conf
-from files.opt.seagate.sspl.setup.setup_error import SetupError
 from files.opt.seagate.sspl.setup.setup_logger import logger
 
+
 class HardReset:
```

Comment:
HardReset needs to be renamed Reset alone, even though actions are related to hard reset. as no more hard/soft reset concept applied now in R2

Reply:
Ack.
Class name changed from HardReset to Reset.